var/spack/repos/builtin/packages/py-diameter-synthesis/package.py

```diff
@@ -0,0 +1,21 @@
+from spack import *
```

Comment:
standard copyright header missing?

Reply:
My bad, I was thinking that the standard copyright didn't apply to new files used for internal packages. I'll add it back.